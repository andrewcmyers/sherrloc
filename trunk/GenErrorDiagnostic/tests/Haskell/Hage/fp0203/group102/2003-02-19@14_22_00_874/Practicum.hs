module Practicum where
eqList        :: (a -> a -> Bool) -> [a] -> [a] -> Bool
eqList = undefined
eqString      :: String -> String -> Bool 
eqString = undefined
kolomBr :: [String] -> Int
kolomBr (x:xs) |eqList eqString xs [] =  length x
               |length x < length (head xs) = kolomBr xs
               |otherwise = kolomBr (x:(drop 1 xs))
kolomBr [] = 0
type Table = [[String]]
neemKolom :: Table -> Int -> [String]
neemKolom (x:xs) k = x!!k-1 : neemKolom xs k
neemKolom [] _= []
-- k-1 should be (k-1)
-- 13,25-27 
