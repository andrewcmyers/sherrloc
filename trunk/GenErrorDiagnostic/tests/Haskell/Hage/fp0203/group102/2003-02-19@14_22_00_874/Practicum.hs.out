Compiling Practicum.hs
(13,26)-(13,27): Type error in infix application
 expression       : x !! k - 1
 operator         : -
   type           : a      -> a   -> a     
   does not match : String -> Int -> String

Compilation failed with 1 error
